partner_compassion/views/partner_compassion_view.xml

```diff
@@ -243,35 +234,24 @@
         <field name="inherit_id" ref="base.view_res_partner_filter"/>
         <field name="arch" type="xml">
             <field name="category_id" position="after">
-                <field name="lang"/>
-                <field name="city" filter_domain="['|',('city','ilike', self),('zip','ilike',self)]"/>
+                <field name="lang" />
+                <field name="city" filter_domain="['|',('city','ilike', self),('zip','ilike',self)]" />
             </field>
             <xpath expr="//group">
                 <filter string="Language" context="{'group_by': 'lang'}"/>
             </xpath>
             <field name="name" position="after">
-                <field name="advocate_details_id"
-                       filter_domain="[('advocate_details_id.engagement_ids', 'ilike', self)]"/>
+                 <field name="advocate_details_id" filter_domain="[('advocate_details_id.engagement_ids', 'ilike', self)]"/>
             </field>
             <!-- Remove filtering of contact people on the customer filter -->
             <filter name="customer" position="attributes">
                 <attribute name="domain">[('customer','=',1)]</attribute>
             </filter>
             <filter name="supplier" position="after">
-                <separator/>
-                <filter string="Sponsor" name="sponsor"
-                        domain="['|', ('category_id.name', '=', 'Sponsor'), ('category_id.name', '=', 'Parrain')]"/>
-                <separator/>
-                <filter string="Advocate" name="advocate"
-                        domain="[('advocate_details_id.state', 'in', ['new', 'active'])]"/>
-                <separator/>
-                <filter string="EN living in DE" name="en_DE"
-                        domain="[('lang', '=', 'en_US'), ('zip_id.lang', '=', '1')]"/>
-                <filter string="EN living in FR" name="en_FR"
-                        domain="[('lang', '=', 'en_US'), ('zip_id.lang', '=', '2')]"/>
-                <filter string="EN living in IT" name="en_IT"
-                        domain="[('lang', '=', 'en_US'), ('zip_id.lang', '=', '3')]"/>
-                <separator/>
+                <separator />
```

Comment:
This change is producing your bug in Travis. It is probably due to a wrong merge during rebase.